mindsdb/utilities/functions.py

```diff
@@ -95,3 +96,9 @@ def get_versions_where_predictors_become_obsolete():
 
     versions_for_updating_predictors = [x for x in versions_for_updating_predictors if len(x) > 0]
     return True, versions_for_updating_predictors
+
+
+def init_lexer_parsers():
+    get_lexer_parser('mindsdb')
+    get_lexer_parser('mysql')
+    get_lexer_parser('sqlite')
```

Comment:
We can remove sqlite and use 'mindsdb' or 'mysql' dialect in mindsdb.api.http.namespaces.analysis.QueryAnalysis.post